### DISCUSSION TITLE
textarea 组件

### DISCUSSION THREAD

**Discussion by yangxiaolu1993** [jdf2e/nutui-react#8] [General]
1、使用 change 事件进行对 value 赋值，输入内容出错

`<TextArea defaultValue={value1} change={change} />
     const [value1, UpdateValue1] = useState(''）
    const change = (value: string, event: Event) => {
      UpdateVale1(value)
    }
`

![image](https://user-images.githubusercontent.com/32215990/138284838-edf5ba99-3e61-4167-8f8c-83f4fa3f2778.png)

2、新增 emit 事件 demo

3、maxLength 设置为 ‘-1’ 时，则失效

![image](https://user-images.githubusercontent.com/32215990/138284938-229add9c-8f49-466e-9cc3-4c0626a53278.png)
 
4、defaultValue 设置的初始值的长度大于 maxlength 设置的长度

![image](https://user-images.githubusercontent.com/32215990/138285040-5821e018-5ba6-43d2-808d-a192aab9528e.png)

 `<TextArea defaultValue={value2} limitshow maxlength="2" />
const [value2, UpdateValue2] = useState('异步文本')`

**Comment by vickyYE** [comment]
已修改完成

**Comment by xiaoyatong** [comment]
![image](https://user-images.githubusercontent.com/84436086/139442357-a8abca5c-fd58-4be7-82b9-1a5ba80a377b.png)

**Reply by vickyYE** [reply]
已修复，更新版本后可查看
